core-request.html

```diff
@@ -64,7 +71,16 @@
        */
       completes: {
         type: Promise,
-        readOnly: true
+        readOnly: true,
+        notify: true,
+        value: function() {
+          var completes = new Promise(function (resolve, reject) {
```

Comment:
Noting here that IE10 does not have native promises, so this would put a dependency on the Promise polyfill (which is not included in webcomponents.js and we have been trying to avoid adding) for IE10 support.  Was this considered, and is it required (given the 0.5 core-ajax didn't have this dependency)?